lib/resources/example_json/r4_examples_communication.rb

```diff
@@ -94,5 +94,56 @@ module Resources
       ]
     }
 
+    R4_COMMUNICATION_CREATE ||= {
```

Comment:
I beautified this. Try it out:

```
{
  "resourceType": "Communication",
  "extension": [
    {
      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/reply-to",
      "valueReference": {
        "reference": "Group/6345478364"
      }
    }
  ],
  "status": "completed",
  "category": [
    {
      "coding": [
        {
          "system": "http://terminology.hl7.org/CodeSystem/communication-category",
          "code": "reminder"
        }
      ]
    }
  ],
  "priority": "urgent",
  "subject": {
    "reference": "Patient/234455"
  },
  "topic": {
    "text": "CDS Query"
  },
  "encounter": {
    "reference": "Encounter/87648476"
  },
  "recipient": [
    {
      "reference": "Practitioner/746484674"
    },
    {
      "reference": "Group/6345478364"
    }
  ],
  "sender": {
    "reference": "Practitioner/3464537365"
  },
  "payload": [
    {
      "contentAttachment": {
        "contentType": "text/plain",
        "data": "Y29tbXVuaWNhaXRvbkA="
      }
    }
  ]
}
```

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/5f8155b282ac3334f2e99e259ec6935aef6a7640